.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main  
+  
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDTAGS: "include_oss include_gcs"
+      CGO_ENABLED: 1
+      GO111MODULE: "auto"
+      GOPATH: ${{ github.workspace }}
+      GOOS: linux
+      
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: src/github.com/docker/distribution
+        fetch-depth: 50
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.*
+
+    - name: Dependencies
+      run: |
+          sudo apt-get -q update
+          sudo -E apt-get -yq --no-install-suggests --no-install-recommends install python-minimal
+          cd /tmp && go get -u github.com/vbatts/git-validation
+          
+    - name: Build
+      working-directory: ./src/github.com/docker/distribution
+      run: |
+        # DCO_VERBOSITY=-q script/validate/dco
```

Comment:
I am not sure exactly what this does and it did cause the CI to fail so I commnted it out.

Reply:
I believe it validates if the commits are signed. We could use [probot](https://probot.github.io/apps/dco/) for this. Not sure who has perms for this to add it in.

Reply:
I think it is fine to have this in here if that is what it does.  However, it failed as it appears there are commits that are not signed.  We could also use branch protection rules to enforce that.

Reply:
If we add the probot, I think we might as well remove that line completely.